fdbserver/RestoreMaster.actor.cpp

```diff
@@ -276,15 +277,17 @@ ACTOR static Future<Version> processRestoreRequest(Reference<RestoreMasterData>
 	self->initBackupContainer(request.url);
 
 	// Get all backup files' description and save them to files
-	wait(collectBackupFiles(self->bc, &rangeFiles, &logFiles, cx, request));
+	Version targetVersion = wait(collectBackupFiles(self->bc, &rangeFiles, &logFiles, cx, request));
+	ASSERT(targetVersion > 0);
 
 	std::sort(rangeFiles.begin(), rangeFiles.end());
 	std::sort(logFiles.begin(), logFiles.end(), [](RestoreFileFR const& f1, RestoreFileFR const& f2) -> bool {
 		return std::tie(f1.endVersion, f1.beginVersion, f1.fileIndex, f1.fileName) <
 		       std::tie(f2.endVersion, f2.beginVersion, f2.fileIndex, f2.fileName);
 	});
 
-	self->buildVersionBatches(rangeFiles, logFiles, &self->versionBatches); // Divide files into version batches
+	self->buildVersionBatches(rangeFiles, logFiles, &self->versionBatches,
+	                          targetVersion); // Divide files into version batches
```

Comment:
Can you move the comment above this line so that the param does not have to be a separate line?

Reply:
Sure.